gosushi_server/spec/tests/calculatePoints.test.js

```diff
@@ -530,9 +590,187 @@ describe('calculateSpecialPoints', function () {
     game.players.forEach(p => p.points = 0);
   })
 
+  const removeNigiriPoints = () => {
+    game.players[1].points -= 2;
+    game.players[2].points -= 3;
+    game.players[3].points -= 3;
+  }
+
   it('should give Wasabi points correctly', () => {
```

Comment:
This is a pretty useless test name haha - maybe specify (if its short) how wasabi points are calculated?

Reply:
The thing is, I'm usually doing a couple vague checks at the same time so idk if we want to specify all of them (each player's hand is a different check). For this one in particular, it's literally not doing any computations - just straight up transferring the data from the card into the player's points. I'm open to suggestions on how to name it better :P